docs/source/backends/spark.rst

```diff
@@ -42,11 +42,11 @@ Use ``ibis.spark.connect`` to create a client.
 
 PySpark client (Experimental)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-.. currentmodule:: ibis.pyspark.api
+.. currentmodule:: ibis.backends.pyspark
 
-The PySpark client is accessible through the ``ibis.pyspark`` namespace.
+The PySpark client is accessible through the ``ibis.backends.pyspark`` namespace.
```

Comment:
Can you revert this please? It's a bit confusing, but this is not the pyspark module, it shouldn't be changed.